selfdrive/test/process_replay/test_debayer.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3
+import os
+import sys
+import subprocess
+import bz2
+import numpy as np
+
+from common.basedir import BASEDIR
+from selfdrive.hardware import PC, TICI
+from selfdrive.test.openpilotci import BASE_URL, get_url
+from selfdrive.version import get_commit
+from tools.lib.logreader import LogReader
+from tools.lib.filereader import FileReader
+
+if TICI:
+  os.environ["PYOPENCL_CTX"] = "0"
+
+try:
```

Comment:
Maybe put this in a different script we can call from the Jenkinsfile before running the test. Don't like running this on import time.